blocks/article-body-block/themes/news.json

```diff
@@ -301,7 +312,10 @@
 			"default": {
 				"box-shadow": "(inset 0 -5px 0 0 var(--border-color), inset 0 5px 0 0 var(--border-color))",
 				"gap": "var(--global-spacing-4)",
-				"padding": "var(--global-spacing-6) 0 var(--global-spacing-5)",
+				"padding-block-start": "var(--global-spacing-6) 0",
+				"padding-inline-end": "var(--global-spacing-5)",
+				"padding-block-end": "var(--global-spacing-6) 0",
+				"padding-inline-start": "var(--global-spacing-5)",
```

Comment:
Here's is the doc for padding: https://developer.mozilla.org/en-US/docs/Web/CSS/padding

"padding-block-start": "var(--global-spacing-6) ",
"padding-inline-end": "0",
"padding-block-end": "var(--global-spacing-5)",
"padding-inline-start": "0",

Reply:
good catch, fixed!